compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1492,6 +1492,18 @@ public BType createInvokableType(List<? extends BLangVariable> paramVars,
 
         boolean foundDefaultableParam = false;
         List<String> paramNames = new ArrayList<>();
+        if (Symbols.isFlagOn(flags, Flags.ANY_FUNCTION)) {
+            BInvokableType bInvokableType = new BInvokableType(paramTypes, null, symTable.anyOrErrorType, null);
+            bInvokableType.flags = flags;
+            BInvokableTypeSymbol tsymbol = Symbols.createInvokableTypeSymbol(SymTag.FUNCTION_TYPE, flags,
+                                                                             env.enclPkg.symbol.pkgID, bInvokableType,
+                                                                             env.scope.owner, location, SOURCE);
+            tsymbol.params = params;
+            tsymbol.restParam = null;
+            tsymbol.returnType = symTable.anyOrErrorType;
```

Comment:
Do we need to set the return type here and in the type in L1495. Isn't it consistent to keep it as null too?

Reply:
Changed.